sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -8,14 +8,14 @@
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
     <DefineConstants>$(DefineConstants);AZURE_NULLABLE;HAS_INTERNALS_VISIBLE_CORE</DefineConstants>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net461;net472;netcoreapp2.1;net5.0;net6.0</TargetFrameworks>
+
+    <!-- TODO: is it really ok to drop support for netcoreapp2.1? -->
```

Comment:
This will need to be resolved before merging to main.

Reply:
Actually, @m-nash, you should be able to revert this whole file, I think.  If you think you need it, let's chat about the reason, because I just made these changes in the feature branch to enable the dependency on the ClientModel library, but you're not adding any dependency in Azure.Core on ClientModel as part of this PR, correct?